honeybee_energy/ventcool/afn.py

```diff
@@ -0,0 +1,391 @@
+# coding=utf-8
+"""Functions to generate airflownetwork for a list of rooms."""
+from __future__ import division
+import math
+
+from .crack import AFNCrack
+from .opening import VentilationOpening
+from .crack_data import crack_data_dict
+
+
+# TODO: Move to ladybug_geometry.Base2DIn3D?
```

Comment:
Yes, we definitely should or it should be in honeybee-core much like this exact same method currently exists in dragonfly-core:
https://github.com/ladybug-tools/dragonfly-core/blob/master/dragonfly/_base.py#L82-L114

Whatever the case, it should not be a publicly-exposed method because it's not going to live in this spot for long and we don't want other developers using it. So please change them to `_compute_bounding_box_x` and `_compute_bounding_box_y` for now.